github/billing.go

```diff
@@ -63,6 +63,46 @@ type AdvancedSecurityCommittersBreakdown struct {
 	LastPushedDate *string `json:"last_pushed_date,omitempty"`
 }
 
+// UsageReportOptions specifies optional parameters for the enhanced billing platform usage report.
+type UsageReportOptions struct {
+	// If specified, only return results for a single year. The value of year is an integer with four digits representing a year. For example, 2025. Default value is the current year.
+
```

Comment:
Please remove a newline between the comment and the field:
```suggestion
	// If specified, only return results for a single year. The value of year is an integer with four digits representing a year. For example, 2025.
	// Default value is the current year.
```

Reply:
This increases readability for [pkg doc](https://pkg.go.dev/github.com/google/go-github/v73/github).

Before:

<img width="970" alt="image" src="https://github.com/user-attachments/assets/5e61677b-c8f2-4b70-b280-4da03dd69ddc" />


After:

<img width="1162" alt="image" src="https://github.com/user-attachments/assets/e665da05-e17a-49f4-9acf-21da9685a177" />